libs/math/tests/unit/distance/entropy_tests.cpp

```diff
@@ -48,5 +43,9 @@ TYPED_TEST(EntropyTest, entropy)
   A.Set(SizeType{2}, DataType(0.3));
   A.Set(SizeType{3}, DataType(0.4));
```

Comment:
I know these are just tests, but shouldn't it be static_cast<DataType>(0.3)?

Reply:
in this case it's a DataType constructor call, so it should be equivalent.